tests/unit/test_cert.py

```diff
@@ -344,8 +344,28 @@ def test_cert_register_multiple_certs_for_auth_key(self):
                 cert_controller.register()
 
                 out, err = self.capsys.readouterr()
-                assert out.count("Multiple registrable certificates for key") > 0
+                assert out.count("Multiple certificates to 'REGISTER' for key") > 0
 
                 with self.capsys.disabled():
                     sys.stdout.write(out)
                     sys.stderr.write(err)
+
+    def test_cert_activate(self):
```

Comment:
Please also test failures/exceptions in addition to the happy path.

Reply:
Any specific scenario do in mind here, most failures are tested already?

Reply:
> Please also add integration tests for "cert import", "cert register" and "cert activate".

I would rather not tackle this until the Jenkins access is sorted out, running them locally is far too time consuming.

Reply:
> Integration tests fail. Failure is related to PyLint errors, which should be fixed.
Most of these PyLint errors are there for a while and are quite absurd, do you think it makes any sense to add ignores for them?

As for integration tests failing -- linting just should not fail integration tests, they should be in separate targets.

Reply:
> Any specific scenario do in mind here, most failures that I could see are tested already?

What if these throw an exception: 

        token_cert_api.activate_certificate(activatable_cert.certificate_details.hash) # responseless PUT
        cert_actions = token_cert_api.get_possible_actions_for_certificate(activatable_cert.certificate_details.hash)

Reply:
There is no specific handler for these, so nothing specific to test.